extensions/applicationinsights-cfgsync-js/src/CfgSyncPlugin.ts

```diff
@@ -121,6 +122,7 @@ export class CfgSyncPlugin extends BaseTelemetryPlugin implements ICfgSyncPlugin
                 _self._addHook(onConfigChange(config, () => {
                     let ctx = createProcessTelemetryContext(null, config, core);
                     _extensionConfig = ctx.getExtCfg(identifier, _defaultConfig);
+                    _blockCdn = !!_extensionConfig.blockCdn;
```

Comment:
maybe it would be better:
```
let preBlockCdn =  _blockCdn;
 _blockCdn = !!_extensionConfig.blockCdn;
if (preBlockCdn !== _blockCdn) {
    if (!_blockCdn) {
        do _fetch....
    } else {
       do clearTimer()
    }
}


```